### ISSUE TITLE
Feature Request: Volumetric Calculator

### ISSUE THREAD

**Issue by TechJed** [CompositeLight/currentRMS-helper#28]
Hirehop have just brought out a Volumetric calc which would be great for working out which vehicle you need. Would need to be done with custom fields, and a second one that is an Uncased tickbox, as you'd just want to calculate cases, not cable that go in cases

**Comment by CompositeLight** [comment]
Yeah, I saw that. I can see this working in Current by adding W, H, and D custom fields (or just one overall volume, but I would include all three so as to ensure the longest thing can fit in a given vehicle) and then creating a custom doc. You wouldn't need the "uncased" tickbox if you just put things in a container that are cased. Since this info would be pulled through the API with other product info it could be used in the GUI the same as weight is.

However, to be honest I'm generally not sold on the idea in general. Basic volumetric info doesn't tell you what can't be tipped, or stacked which has a big effect on 'real' volume used. If you're just dealing with big cases going into trailers then I guess it kinda works, but for anything else the weight is going to limit you far sooner than volume will.

**Comment by ClearsoundPro** [comment]
I saw this update too, I do agree with Gareth, weight is key, I can judge transport requirements by weight more than what the volumetric info is. I never worry about whether it fits, I worry more about if it’s too heavy.

**Comment by TechJed** [comment]
We do a fair amount of jobs with set, where space becomes an issue before weight, so it would be useful to see whether you should send an 18T, Urban Trailer or Megacube. Might be something we look at, as tbs we could incorporate into documents without needing to put in the plugin

**Comment by CompositeLight** [comment]
Ah, truck full of set - got it. Given how organisation specific this would need to be (what vehicles you have etc) I think you're right that it makes sense to do it within docs so you can build in as many parameters as you need. Good luck!
